Makefile

```diff
@@ -41,5 +41,4 @@ deploy: ## Deploys the theme on a host server
 	rsync --recursive --delete asset/img/ /var/www/html/themes/mitlibraries-theme-omeka/asset/img/
 	rsync --recursive --delete asset/js/ /var/www/html/themes/mitlibraries-theme-omeka/asset/js/
 	rsync --recursive --delete config/ /var/www/html/themes/mitlibraries-theme-omeka/config/
-	rsync --recursive --delete helper/ /var/www/html/themes/mitlibraries-theme-omeka/helper/
```

Comment:
InfraEng was tagged on this PR because of the `Makefile` ownership in the `CODEOWNERS` file. I'm not going to approve / no approve, but I will note that on behalf of InfraEng, I'm okay with this change to the Makefile.

Reply:
I initially tried this PR without the change, but that causes the deploy to fail because the directory doesn't exist anymore.